include/lsst/jointcal/Projectionhandler.h

```diff
@@ -33,14 +33,14 @@ class IdentityProjection : public ProjectionHandler
 };
 
 //! A possible implementation of ProjectionHandler for fitting actual WCS's.
-/* ! We arbitrarily chose that all chips from a same shoot have the same tangent point, but other choices can be made. */
-class OneTPPerShoot : public ProjectionHandler
+/* ! We arbitrarily chose that all chips from a same visit have the same tangent point, but other choices can be made. */
+class OneTPPerVisit : public ProjectionHandler
```

Comment:
I suggest `OneTPPerVisitHandler` or some similar way of expressing that this is a subclass of `ProjectionHandler`. The doc would be clearer as "A projection handler in which all CCDs from the same visit have the same tangent point". The "other choices can be made" seems unnecessary.